views/waiting_room_game.erb

```diff
@@ -0,0 +1,56 @@
+
+<style>
+  .centered {
+
+    /* Gets pos of middle of the screen, no matter the size or scroll. */
+    position: absolute;
+
+    /* Set Margain */
+    top: 50%;
+    left: 50%;
+
+    /* This moves the objects */
+    transform: translate(-50%, -50%);
+    text-align: center;
+  }
+
+  .off_centered {
+
+    /* Gets pos of middle of the screen, no matter the size or scroll. */
+    position: absolute;
+
+    /* Set Margain */
+    top: 35%;
+    left: 50%;
+
+    /* This moves the objects */
+    transform: translate(-50%, -50%);
+    text-align: center;
+  }
+</style>
+
+
+<!-- Refresh every second -->
+<meta http-equiv="refresh" content="2">
```

Comment:
Really interesting and clear way of refreshing the page, I didn't know you could do this!!